Code/GraphMol/Aromaticity.cpp

```diff
@@ -361,17 +366,27 @@ void applyHuckelToFused(
   unsigned int i, curSize = 0;
   INT_VECT comb;
   pos = -1;
+
+  size_t nRingBonds;
```

Comment:
wouldn't it be a lot more efficient to do this with a dynamic_bitset to track which bonds have been seen and then calling count() at the end?

Reply:
ah, probably true. I'll update.

Reply:
Although setting and getting values in a bitset is not so so fast, because of all the bitmasks. Looks like the unordered_set approach is about 1.2x slower for this portion of the call.